internal/providers/terraform/azure/testdata/bastion_host_test/bastion_host_test.golden

```diff
@@ -0,0 +1,13 @@
+
+ Name                                            Monthly Qty  Unit   Monthly Cost 
+                                                                                  
+ azurerm_bastion_host.example                                                     
+ ├─ Azure Bastion (Basic)                                730  hours       $138.70 
```

Comment:
Typically we leave 'Azure' out of the name to avoid redundancy.
```suggestion
 ├─ Bastion instance (Basic)                                730  hours       $138.70 
```

Reply:
Done. I took the liberty to use Bastion host instead of Bastion instance for the name as it is the name of the tf resource.